pkgs/development/tools/build-managers/cmake/default.nix

```diff
@@ -58,6 +61,7 @@ stdenv.mkDerivation rec {
       "--no-system-jsoncpp"
     ]
     ++ optional (!stdenv.isCygwin) "--system-libs"
```

Comment:
Oh you mean for things built with cmake? Not sure, but it was my understanding these libraries would be used to implement cmake functionality itself, not to be used for building/linking other things... but maybe that's not so.
Do you know? As you say, those should be explicitly listing the libraries they need via buildInputs.